src/secp256k1/README.md

```diff
@@ -1,7 +1,7 @@
 libsecp256k1
 ============
 
-[![Build Status](https://travis-ci.org/bitcoin-core/secp256k1.svg?branch=master)](https://travis-ci.org/bitcoin-core/secp256k1)
+[![Build Status](https://api.cirrus-ci.com/github/bitcoin-core/secp256k1.svg?branch=master)](https://cirrus-ci.com/github/bitcoin-core/secp256k1)
 
```

Comment:
```suggestion
```
This status is misleading for this copy of the source.

Reply:
I don't think this matters, and I'd rather keep the subtree diff from upstream to the minimum required.